### DISCUSSION TITLE
Is this safe to use for newsletters?

### DISCUSSION THREAD

**Discussion by putzwasser** [disposable/disposable#118] [Q&A]
Hi,

i found your list of disposable email addresses. I like the idea. I'd like to incorporate it into my newsletter to prevent signups from fake email addresses.

I have one question though: Is this purely for blocking fake/spam email addresses? Or is this list for **all** disposable email addresses?

By all disposable email addresses I mean services like yahoo, gmail, and the like.

I'm asking because I found a list, which included those free email provider, too. As my newsletter is B2C I don't want to exclude those free email provider.

**Comment by smeinecke** [comment]
Thank you for your question. We try to include only domains that offer temporary email addresses - freemailers do not count in this case, even if they unfortunately also offer temporary email addresses.

Since we mostly only combine external lists and only crawl the addresses of a few providers ourselves, it can of course occasionally happen that freemailers also appear on the list (has unfortunately already happened in the past). Many of them are already in the https://github.com/disposable/disposable/blob/master/whitelist.txt and are automatically filtered out - but of course there can always be false positives. 

If you decide to use the list and you get feedback about wrong hosts, you are very welcome to open a ticket, so that we can remove them from the list asap.
